examples/guestbook/README.md

```diff
@@ -383,3 +383,60 @@ kubectl delete -f examples/guestbook/redis-slave-service.json
 kubectl delete pod redis-master # This is the only pod that requires manual removal.
 ```
 
+### Troubleshooting
+
+Guestbook can go wrong for a variety of reasons, which makes it an effective test.  Lets test the web app simply using *curl*, so we can see whats going on.
+
+To post a message (Note that this call *overwrites* the messages field), so it will be reset to just one entry.
+
+```
+curl "localhost:8000/index.php?cmd=set&key=messages&value=jay_sais_hi"
+```
+
+And, to get messages afterwards...
+
+```
+curl "localhost:8000/index.php?cmd=get&key=messages"
+```
+
+1) When the *Web page hasn't come up yet*:
+
+When you go to localhost:8000, you might not see the page at all.  Testing it with curl...
+```shell
+   ==> default: curl: (56) Recv failure: Connection reset by peer
+```
+
+This means the web frontend isn't up yet.  Wait a while, possibly about 2 minutes or more, depending on your set up. Also, run a *watch* on docker ps, to see if containers are cycling on and off or not starting.
+
+2) *When Redis can't connect to master*:
+
+```shell
+<b>Fatal error</b>:  Uncaught exception 'Predis\Connection\ConnectionException' with message 'php_network_getaddresses: getaddrinfo failed: Name or service not known [tcp://:0]' in /vendor/predis/predis/lib/Predis/Connection/AbstractConnection.php:141
+```
+
+The fix: Make sure that environmental variables are being set correctly. 
```

Comment:
Give an example of which env vars you're talking about? (maybe the ones that weren't being set when you ran this?)